orchestration/dagster_orchestration/pytest.ini

```diff
@@ -3,3 +3,6 @@ console_output_style = progress
 env_files =
 	.env.test
 markers = e2e: marks tests as an end-to-end test (deselect with '-m "not e2e"')
+
+# skip e2e tests by default
+addopts = -m "not e2e"
```

Comment:
By default, we'll only run the e2e suite in CI, but it's still invokable locally.